package.json

```diff
@@ -8,6 +8,7 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
+    "nodemon": "^1.19.0",
```

Comment:
Is there a reason for this dependency?

Reply:
No, it's just for me to run the application.

I am sorry for this, I'm new to GitHub. Should I remove this?
I also made a mistake with a closing bracket and I'm trying to to update with the fix

Reply:
Yes, remove it

Reply:
done